packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -655,11 +703,7 @@ export function GraphiQLInterface(props: GraphiQLInterfaceProps) {
                   <div ref={editorToolsResize.secondRef}>
                     <section
                       className="graphiql-editor-tool"
-                      aria-label={
-                        activeSecondaryEditor === 'variables'
-                          ? 'Variables'
-                          : 'Headers'
-                      }
```

Comment:
I replaced this ternary with a basic method. I tried to not be clever and go for a plain method rather than a .capitalize for the tab name